pegasus/sites.v3/code.org/views/homepage_below_hero_pd_banner.haml

```diff
@@ -0,0 +1,36 @@
+- if Homepage.show_professional_learning_banner(request)
```

Comment:
Since this banner is shown in more places than just the homepage, we should probably rename this. (Maybe just professional_learning_marketing_banner or something like that)